packages/phrases/src/locales/fr/translation/admin-console/general.ts

```diff
@@ -0,0 +1,40 @@
+const general = {
+  placeholder: 'Placeholder',
```

Comment:
This is not an oversight. In French, we don't have any equivalent words to my knowledge, so I leave the default term. I will propose a new translation if I find a better alternative.